packages/ui/src/components/table.tsx

```diff
@@ -43,7 +43,7 @@ TableHeader.displayName = 'TableHeader'
 
 const TableBody = forwardRef<HTMLTableSectionElement, HTMLAttributes<HTMLTableSectionElement>>(
   ({ className, ...props }, ref) => (
-    <tbody ref={ref} className={cn('[&_tr:last-child]:border-0', className)} {...props} />
+    <tbody ref={ref} className={cn('[&_tr:last-child]:border-0 [&>tr:hover]:bg-background-4', className)} {...props} />
```

Comment:
To me a change of colour on hover denotes that something is clickable. This might be the ideal for the usage in the labels work, but I don't think we can make that call for _all_ tables. Can we make this opt-in as, perhaps, an extra prop?